test/functionalTest/cases/117_convop_using_standard_ops/postContextEntitiesByEntityId.test

```diff
@@ -0,0 +1,332 @@
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+ConvOp postContextEntitiesByEntityId: POST /v1/registry/contextEntities/{entityId::id}
+
+--SHELL-INIT--
+dbInit CB
+dbInit CP1
+brokerStart CB
+brokerStart CP1
+
+--SHELL--
+
+#
+# 01. Register E1 using POST /v1/registry/contextEntities/E1
+# 02. Discover E1
+# 03. Update the registration of E1, CP1 as providing application, using postEntityByIdAttributeByName
+# 04. Discover E1
+# 05. Create E1/A1 in CP1
+# 06. Update/UPDATE E1/A1 in CB
+# 07. Query E1/A1 in CB
+#
+
+echo "01. Register E1 using POST /v1/registry/contextEntities/E1"
+echo "=========================================================="
+payload='{
+  "duration": "PT1M", 
+  "providingApplication": "none"
+}'
+orionCurl --url /v1/registry/contextEntities/E1 --payload "$payload" --json
+REG_ID=$(echo "$_response" | grep -Po '(?<="registrationId" : ")[^"]*')
+echo
+echo
+
+
+echo "02. Discover E1"
+echo "==============="
+orionCurl --url /v1/registry/contextEntities/E1 --json
+echo
+echo
+
+
+echo "03. Update the registration of E1, CP1 as providing application, using postEntityByIdAttributeByName"
+echo "===================================================================================================="
+payload='{
+  "duration": "PT1M", 
+  "providingApplication": "http://localhost:'$CP1_PORT'/v1", 
+  "registrationId": "'$REG_ID'"
+}'
+orionCurl --url /v1/registry/contextEntities/E1/attributes/A1 --payload "$payload" --json
+echo
+echo
+
+
+echo "04. Discover E1"
+echo "==============="
+orionCurl --url /v1/registry/contextEntities/E1 --json
+echo
+echo
+
+
+echo "05. Create E1/A1 in CP1"
+echo "======================="
+payload='{
+  "contextElements": [
+    {
+      "type": "",
+      "isPattern": "false",
+      "id": "E1",
+      "attributes": [
+        {
+          "name": "A1",
+          "type": "degree",
+          "value": "20"
+        }
+      ]
+    }
+  ],
+  "updateAction": "APPEND"
+}'
+orionCurl --url /v1/updateContext --payload "$payload" --port $CP1_PORT --json
+echo
+echo
+
+
+echo "06. Update/UPDATE E1/A1 in CB"
+echo "============================="
+payload='{
+  "contextElements": [
+    {
+      "type": "",
+      "isPattern": "false",
+      "id": "E1",
+      "attributes": [
+        {
+          "name": "A1",
+          "type": "degree",
+          "value": "20"
```

Comment:
Should be different from the 20 used in the original creation to being able to know if the update has been actually done?